services/freelance-matching/.env.example

```
@@ -9,6 +9,7 @@ REDIS_PASSWORD=
 # API Keys
 OPENAI_API_KEY=your_openai_api_key
 HUGGINGFACE_API_KEY=your_huggingface_api_key
+HF_API_KEY=your_huggingface_api_key # Used by the low-cost embedding service
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HF_API_KEY key should go before the HUGGINGFACE_API_KEY key

(UnorderedKey)